src/chrome/content/rules/Status.io.xml

```diff
@@ -0,0 +1,11 @@
+<!--developers. timed out-->
```

Comment:
It seems that some customers get their own subdomain, which are not secureable. Can you document that?